### PR TITLE
Task port

### PR DESCRIPTION
**Pull request by ajhswan**
Completed portfolio section of site

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.ds.store
+.ds_store
```
Comment (alexstonelunde) [inline .gitignore:1]: Note sure if this has to be case sensitive: .DS_Store

File: index.html [hunk 2]
```diff
@@ -29,5 +29,22 @@ <h3>executing ideas of the future</h3>
...
+				<p>html and css only project. Created a demo site which inclued a hero image with buttons and a featured products section</p>
+			</div>
+			
+			<div id='box2' class='projects'>
+				<img src='projects.jpg'/>
+				<p>html and css logo. Created a logo using just css and html. Demonstration of how to get mutiple &ltdiv&gt 's to overlap in order to create a vector image.</p>
+			</div>
+			<div id='box3' class='projects'>
+				<img src='projects.jpg'/>
+				<p>JavaScript project. Build a portfolio site using html, csss and javascript</p>
```
Comment (alexstonelunde) [inline index.html]: Typo @ CSS.

File: index.html [hunk 3]
```diff
@@ -29,5 +29,22 @@ <h3>executing ideas of the future</h3>
...
+			</div>
+
```
Comment (alexstonelunde) [inline index.html]: extra line

File: styles.css [hunk 4]
```diff
@@ -100,11 +104,41 @@ li:hover {
 	margin-left: 30px;
 }
 
+section {
+	margin-right: 17.5%;
+	margin-left: 17.5%;
+	height: 800px;
+	background-color: rgb(255,255,255);
+}
+.projects {
```
Comment (alexstonelunde) [inline styles.css]: create a new line.

File: styles.css [hunk 5]
```diff
@@ -100,11 +104,41 @@ li:hover {
 	margin-left: 30px;
 }
 
+section {
+	margin-right: 17.5%;
+	margin-left: 17.5%;
+	height: 800px;
+	background-color: rgb(255,255,255);
+}
```
Comment (alexstonelunde) [inline styles.css:112]: create a new line

File: styles.css [hunk 6]
```diff
@@ -100,11 +104,41 @@ li:hover {
...
+
+#box3 {
+	margin-right: 0;
+}
+
+.projects p {
+	width: 120%;
+	margin-left: 5%;
+}
```
Comment (alexstonelunde) [inline styles.css:128]: new line

**Review by alexstonelunde — CHANGES_REQUESTED** [review]
Code looks really clean, just a couple of spacing inconsistencies and a typo.

**Review by ajhswan — COMMENTED** [review]
Changes made

File: .gitignore (hunk 1 above)
Comment (ajhswan) [reply .gitignore:1]: it works and has removed the file from git so looks like it is not.

**Pull request merged**